syntax/coq.vim

```diff
@@ -259,6 +259,17 @@ syn keyword coqLtac contained abstract constr context end external eval fail fir
 syn keyword coqLtac contained idtac in let ltac lazymatch multimatch match of rec reverse solve type with
 syn match   coqLtac contained "|-\|=>\|||\|\[\|\]\|\<_\>\||"
 
+" Ltac2
+syn region coqLtac2Decl contains=coqLtac2Ident,coqVernacPunctuation,coqLtacContents start="\<Ltac2\%(\_sType\)\?\>" end="\.\_s" keepend
+syn region coqLtac2Decl contains=coqLtac2Eval start="\<Ltac2\_sEval\>" end="\.\_s" keepend
+syn region coqLtac2Decl contains=coqLtac2Notation,coqLtacContents start="\<Ltac2\_sNotation\>" end="\.\_s" keepend
+syn region coqLtac2Decl contains=coqIdent matchgroup=coqVernacCmd start="\<Print\_sLtac2\>" end="\.\_s" keepend
+
+syn region coqLtac2Ident contained matchgroup=coqVernacCmd start="Ltac2\>\%(\_sType\)\?" matchgroup=coqIdent end="[_[:alpha:]][_'[:alnum:]]*"
```

Comment:
Should this be
```suggestion
syn region coqLtac2Ident contained matchgroup=coqVernacCmd start="\<Ltac2\>\%(\_sType\)\?" matchgroup=coqIdent end="[_[:alpha:]][_'[:alnum:]]*"
```

Reply:
No you're right, this might still occur in the middle of the command.